client/signup/steps/design-picker/index.jsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { identity } from 'lodash';
+import DesignPicker from '@automattic/design-picker';
+
+/**
+ * Internal dependencies
+ */
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import StepWrapper from 'calypso/signup/step-wrapper';
+import { submitSignupStep } from 'calypso/state/signup/progress/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class DesignPickerStep extends Component {
+	static propTypes = {
+		goToNextStep: PropTypes.func.isRequired,
+		signupDependencies: PropTypes.object.isRequired,
+		stepName: PropTypes.string.isRequired,
+		locale: PropTypes.string.isRequired,
+		translate: PropTypes.func,
+	};
+
+	static defaultProps = {
+		useHeadstart: true,
+		translate: identity,
+	};
+
+	pickDesign = ( selectedDesign ) => {
+		recordTracksEvent( 'calypso_signup_select_design', {
+			theme: `pub/${ selectedDesign?.theme }`,
+			template: selectedDesign?.template,
+		} );
+
+		this.props.submitSignupStep(
+			{
+				stepName: this.props.stepName,
+				selectedDesign,
+			},
+			{
+				selectedDesign,
+			}
+		);
+
+		this.props.goToNextStep();
+	};
+
+	renderDesignPicker() {
+		// props.locale obtained via `localize` HoC
+		return <DesignPicker locale={ this.props.locale } onSelect={ this.pickDesign } />;
```

Comment:
In order to achieve feature parity, should the designs be randomised?

Currently, in Gutenboarding designs are shuffled and saved in the `onboard` store (see `client/landing/gutenboarding/index.tsx`)

Reply:
Not part of this PR, but need to open a new issue to track it before merging

Reply:
We may do that in the future. We'll have the randomization function in `design-picker` package soon. See https://github.com/Automattic/wp-calypso/issues/51404